CONTRIBUTING.md

```diff
@@ -11,8 +11,8 @@ I'm not very picky about how you should contribute, but I ask that the following
 example of its usage. An optimal submission would also include a short real-world use case for the feature.
 * Make sure the feature is in the correct C++ version.
 * Make sure you've added the feature to the table of contents.
-* Make sure you have also added the feature to the separate major C++ readme files (ie. CPP11.md, CPP14.md, etc.).
+* Make sure you have also added the feature to the separate major C++ readme files (i.e. CPP11.md, CPP14.md, etc.).
 * Keep additions/deletions of content consistent with the cheatsheet's goals.
 
 ## Goals
-My goal for this cheatsheet is to prefer conciseness over absolute completeness. Examples of features should be minimal: if an example is overly complicated, large, or is more of an obscure usage of the feature then it will most likely be rejected in review. The reason for this goal is to teach users what the most popular uses of these features will be, and for a more thorough investigation, to learn about those from external C++ resources.
+My goal for this cheatsheet is to prefer conciseness to absolute completeness. Examples of features should be minimal: if an example is overly complicated, large, or is more of an obscure usage of the feature then it will most likely be rejected in review. The reason for this goal is to teach users what the most popular uses of these features will be, and for a more thorough investigation, to learn about those from external C++ resources.
```

Comment:
The original English is correct here.

Reply:
Good to know, thanks! Reverted such changes.